tests/applepay_test.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime, timedelta
+
+from pytz import utc
+
+from applepay import payment
+from applepay import utils as applepay_utils
+
+import utils as test_utils
+
+
+def test_retrieve_signature_signing_time():
+    # Given a detached CMS signature in the token,
+    token = test_utils.load_json_fixture('tests/fixtures/token.json')
```

Comment:
I see you didn't include the token.json file as part of this PR.  Was that on purpose?  I guess it would contain propriety data.